spec/tren_spec.cr

```diff
@@ -11,4 +12,13 @@ describe Tren do
   it "should overload method" do
     get_users("fatih", 2).should eq("select * from users where name = 'fatih' and age = 2")
   end
+
+  it "should create and use method" do
```

Comment:
Please explain this better. Something like `it "should parse multiple queries from a file"`